src/Curators/DBALNewsArticleRepository.php

```diff
@@ -84,6 +84,17 @@ public function create(NewsArticle $newsArticle): void
             ->execute();
     }
 
+    public function delete(UUID $id): void
+    {
+        $queryBuilder = $this->connection->createQueryBuilder();
```

Comment:
Same here, I think there is a `->delete()` shorthand

Reply:
Same issue with the `delete` method of `Connection` https://github.com/doctrine/dbal/blob/v2.9.3/lib/Doctrine/DBAL/Connection.php#L636